.gitignore

```diff
@@ -63,3 +63,5 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
+.DS_Store
+/JordanResources
```

Comment:
i had all my icon and splashscreen exports here, before i dragged them into the xcode assets. I like to keep them separate just in case i need them in future.